# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1265192848 18000
#      Wed Feb 03 05:27:28 2010 -0500
# Node ID dc088edd9a78816c2219f25964ac03d78a62e0b3
# Parent  874510cea8916d732e50e5931d846e2f04941af8
configure.ac: don't check for raise or strdup

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,12 @@
+2010-02-03  John W. Eaton  <jwe@octave.org>
+
+	* configure.ac: Don't check for raise or strdup.
+
 2010-02-03  John W. Eaton  <jwe@octave.org>
 
 	* configure.ac: Don't check for poll or usleep.
 	* bootstrap.conf (gnulib_modules): Include nanosleep and sleep
 	in the list.
 
 2010-02-03  John W. Eaton  <jwe@octave.org>
 
diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -1488,20 +1488,20 @@ fi
 
 ### Checks for functions and variables.
 
 AC_CHECK_FUNCS(basename canonicalize_file_name \
   chmod dup2 endgrent endpwent execvp expm1 expm1f fcntl fork \
   getegid geteuid getgid getgrent getgrgid getgrnam getpgrp getpid \
   getppid getpwent getpwuid getuid getwd _kbhit kill \
   lgamma lgammaf lgamma_r lgammaf_r localtime_r log1p log1pf \
-  mkstemp pipe putenv raise \
+  mkstemp pipe putenv \
   realpath resolvepath rindex roundl select setgrent setlocale \
   setpwent setvbuf siglongjmp \
-  snprintf strdup strerror strsignal tempnam tgammaf trunc umask \
+  snprintf strerror strsignal tempnam tgammaf trunc umask \
   uname utime vfprintf vsprintf vsnprintf waitpid \
   _chmod _snprintf x_utime _utime32)
 
 AC_LANG_PUSH(C++)
 AC_CHECK_DECLS([exp2, round, tgamma], [], [], [[#include <cmath>]])
 AC_CHECK_FUNCS([exp2 round tgamma])
 AH_VERBATIM([Z_FUNCS_AND_DECLS], [
 #if defined (__cplusplus)
